### PR TITLE
issue #32297 - add metric to control default freeform bill-to setting

### PR DESCRIPTION
**Pull request by gilmoskowitz**

### CODE REVIEW

File: foundation-database/public/functions/convertprospecttocustomer.sql [hunk 1]
```diff
@@ -1,29 +1,23 @@
...
   SELECT * INTO _p
   FROM prospect
-  WHERE (prospect_id=pProspectId);
+  WHERE prospect_id = pProspectId;
 
-  IF (EXISTS(SELECT cust_id FROM custinfo WHERE cust_id=pProspectId)) THEN
-    RAISE EXCEPTION '[xtuple: convertProspectToCustomer, -10]';
+  IF EXISTS(SELECT 1 FROM custinfo WHERE cust_id = pProspectId) THEN
+    RAISE EXCEPTION 'Cannot convert prospect % as there is already a customer with the same id [xtuple: convertProspectToCustomer, -10, %]',
+                    _p.prospect_number, pProspectId;
```
Comment (ChristopherCotnoir) [inline foundation-database/public/functions/convertprospecttocustomer.sql]: The two parameters should be identical.

**Comment by ChristopherCotnoir** [comment]
jenkins - retest this please

**Comment by bendiy** [comment]
jenkins - retest this please

**Comment by bendiy** [comment]
jenkins - retest this please

**Pull request merged**